openslides/assignments/static/js/assignments/site.js

```diff
@@ -128,9 +131,12 @@ angular.module('OpenSlidesApp.assignments.site', [
                 },
                 {
                     key: 'description',
-                    type: 'textarea',
+                    type: 'editor',
```

Comment:
A server side check with `validate_html` is needed.

Reply:
Done (in full and short serializer, right?)